### ISSUE TITLE
if the game doesn't have the file Unity.TextMeshPro.dll it won't be possible to change the game's font?

### ISSUE THREAD

**Issue by nonamebatbai** [bbepis/XUnity.AutoTranslator#192]
if the game doesn't have the file Unity.TextMeshPro.dll it won't be possible to change the game's font?

**Comment by gravydevsupreme** [comment]
Yes, because you just need a version of the Unity Engine and TextMesh Pro that is compatible with the one used by the game when generating the font asset (compatible does not mean the exact same version. It depends on the changes.).

No, because obtaining the older versions of TextMesh Pro is hard because they are no longer for sale. Essentially you'd have to query the TextMesh Pro author for the required tooling in that case.

**Comment by nonamebatbai** [comment]
> Yes, because you just need a version of the Unity Engine and TextMesh Pro that is compatible with the one used by the game when generating the font asset (compatible does not mean the exact same version. It depends on the changes.).
> 
> No, because obtaining the older versions of TextMesh Pro is hard because they are no longer for sale. Essentially you'd have to query the TextMesh Pro author for the required tooling in that case.

Thank you for answering the question
